api/s2n.h

```diff
@@ -436,6 +436,30 @@ extern int s2n_connection_is_session_resumed(struct s2n_connection *conn);
 S2N_API
 extern int s2n_connection_is_ocsp_stapled(struct s2n_connection *conn);
 
+typedef enum {
+    S2N_HANDSHAKE_SIGNATURE_ANONYMOUS,
+    S2N_HANDSHAKE_SIGNATURE_RSA,
+    S2N_HANDSHAKE_SIGNATURE_ECDSA,
+    S2N_HANDSHAKE_SIGNATURE_RSA_PSS_RSAE,
+    S2N_HANDSHAKE_SIGNATURE_RSA_PSS_PSS
+} s2n_handshake_signature_algorithm;
+
+typedef enum {
+    S2N_HANDSHAKE_HASH_NONE,
+    S2N_HANDSHAKE_HASH_MD5,
+    S2N_HANDSHAKE_HASH_SHA1,
+    S2N_HANDSHAKE_HASH_SHA224,
+    S2N_HANDSHAKE_HASH_SHA256,
+    S2N_HANDSHAKE_HASH_SHA384,
+    S2N_HANDSHAKE_HASH_SHA512,
+    S2N_HANDSHAKE_HASH_MD5_SHA1,
+} s2n_handshake_hash_algorithm;
```

Comment:
Same as above, we are not leaking any information here and can expose the internal enum: https://github.com/aws/s2n-tls/blob/fe8df74ddafb712d3141056107db7228b19b6ef7/crypto/s2n_hash.h#L28-L39